spec/system/candidate_interface/carry_over/candidate_carries_over_different_application_states_spec.rb

```diff
@@ -172,9 +172,10 @@ def then_i_am_ask_to_apply_for_courses_into_the_new_recruitment_cycle
   end
 
   def then_i_am_on_the_pages_that_is_possible_to_carry_over_an_application
-    expect(page).to have_current_path(
-      candidate_interface_start_carry_over_path,
-    ).or(have_current_path(candidate_interface_application_complete_path))
+    # rubocop:disable Capybara/CurrentPathExpectation
+    expect(page.current_path).to eq(candidate_interface_start_carry_over_path)
```

Comment:
Weird one....

Reply:
It looks like the gem was providing the functionality for the `.or` now that we've removed it, it causes the tests to fail with: 

```
 NoMethodError:
       undefined method `or' for an instance of Capybara::RSpecMatchers::Matchers::HaveCurrentPath
```
I'd prefer if we re-write the test so that we don't have to disable rubocop in line like that, as if we have to do this, it suggests a wider problem.